Misc/NEWS.d/next/Library/2017-09-10-15-20-14.bpo-31412.lNbm-A.rst

```diff
@@ -0,0 +1,2 @@
+`wave.open()` now accepts PathLike objects. Contributed by Michael Scott
```

Comment:
PathLike -> :class:\`os.PathLike\`

Reply:
Or just `path-like`. Or `:term:'path-like objects <path-like object>'` (use backquotes instead of apostrophes).